modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/BundleInfoImpl.java

```diff
@@ -42,6 +42,9 @@
 
     protected String manifest;
 
+    /**
+     * @since 11.1
+     */
```

Comment:
Could be written this way to minimize space taken, especially on a `protected` member
```suggestion
    /** @since 11.1 */
```

Reply:
:+1: i'll probably do that kind of cleanup on others independently, then